### ISSUE TITLE
ARIA attribute unsupported or prohibited - table wrapper divs

### ISSUE THREAD

**Issue by briand44** [uiowa/uiowa801#9593]
# Motivation

Siteimprove is flagging our table wrapper divs for ARIA attribute unsupported or prohibited.

# Additional Info

Siteimprove example issue - https://my2.siteimprove.com/Inspector/1001224/A11Y/Page?pageId=47299245475&impmd=5B1F30908A90030B86281C1D5C967C29&issue=cantTell+failed&wcag=twopointtwo#/sia-r18/failed/SmZsRSsxckI=;NWZlY2JmMDU=

https://alfa.siteimprove.com/rules/sia-r18

Slack conversation - https://iowaweb.slack.com/archives/C0164FNN1PV/p1768343649661159

https://www.drupal.org/project/responsive_table_filter/issues/3567997

**Comment by briand44** [comment]
## Grooming
 
**Quadrant:** 

Clear

 **Discussion:**
 - patch at https://www.drupal.org/project/responsive_table_filter/issues/3567997 low risk
   - one line code change, nothing stored in db, could later be removed with little pain
 - Bonus to push issue forward on D.O. (Comment and mark as RBTC once tested)

**Proposed solution or definition of done:** 
- Apply patch from https://www.drupal.org/project/responsive_table_filter/issues/3567997
- Verify resolves the ARIA issue and Siteimprove is happy
- Verify no regressions
- Bonus to push issue forward on D.O. (Comment and mark as RBTC once tested)

**Comment by joewhitsitt** [comment]
Can be reproduced with browser extension:

<img width="969" height="762" alt="Image" src="https://github.com/user-attachments/assets/5eabbea5-4398-48b9-87a5-93f94dbaeee0" />
